File: src/main/java/shared/model/Tour.java
package model;

import java.util.List;

/**
 * Created by gzae on 12/13/16.
 */
public class Tour extends Event{

    public String vehicle;
    public int miles;
    public String destination;
    public int emptySeats;
    public List<User> participants;


    public Tour(){
        super();
    }
}

File: src/main/web/src/app/shop/shared/model/tour.ts
import {Party} from "./party";
import {ClubRole} from "../../../shared/model/club-role";
import {EventType, typeToInteger} from "./event-type";

export interface Tour extends Party {
	vehicle: string;
	miles: number;
}

export function createTour(): Tour {
	return {
		id: -1,
		title: "",
		date: new Date(),
		description: "",
		expectedReadRole: ClubRole.Gast,
		expectedCheckInRole: ClubRole.Gast,
		expectedWriteRole: ClubRole.Gast,
		author: [],
		route: [],
		images: ["resources/images/Logo.png"],
		capacity: -1,
		groupPicture: "",
		price: 0,
		emptySeats: -1,
		type: typeToInteger(EventType.tours),
		vehicle: "",
		miles: -1
	}
}
